src/FAudio_internal.c

```diff
@@ -90,7 +90,7 @@ void FAudio_INTERNAL_debug(
 	va_end(va);
 
 	/* Print, finally. */
-	FAudio_Log(output);
+	FAudio_Log(output)
```

Comment:
This is an error, isn't it?

Reply:
Oh hey, turns out it isn't, it's a bug in my macros:

https://github.com/FNA-XNA/FAudio/blob/master/src/FAudio_internal.h#L79
https://github.com/FNA-XNA/FAudio/blob/master/src/FAudio_internal.h#L140

Reply:
Happy to help find bugs :)